index.js

```diff
@@ -41,87 +43,79 @@ const displayGuestScore = (score) => (guestPoint.textContent = score);
 
 const showMessage = (msgEl, msgVal) => (msgEl.textContent = msgVal);
 
+function showResetButton() {
+  startButton.style.display = "none";
+  resetButton.style.display = "block";
+}
 function eachRoundScoreDisplay (roundVal, hScore, gScore) {
   let rVal = roundVal;
-  if (rVal == 1){
+  if (rVal === 1){
     homeRound1Score.textContent = hScore;
     guestRound1Score.textContent = gScore;
   }
-  if (rVal == 2){
+  if (rVal === 2){
     homeRound2Score.textContent = hScore;
     guestRound2Score.textContent = gScore;
   }
-  if (rVal == 3){
+  if (rVal === 3){
     homeRound3Score.textContent = hScore;
     guestRound3Score.textContent = gScore;
   }
-  if (rVal == 4){
+  if (rVal === 4){
     homeRound4Score.textContent = hScore;
     guestRound4Score.textContent = gScore;
     calculateFinalScore();
-  }  
+  } 
 }
 
 function calculateFinalScore() {
   hFinalScore = parseInt(homeRound1Score.textContent) + parseInt(homeRound2Score.textContent) + parseInt(homeRound3Score.textContent) + parseInt(homeRound4Score.textContent);
   gFinalScore = parseInt(guestRound1Score.textContent) + parseInt(guestRound2Score.textContent) + parseInt(guestRound3Score.textContent) + parseInt(guestRound4Score.textContent);
+  
   totalHomeScore.textContent = hFinalScore;
   totalGuestScore.textContent = gFinalScore;
-  if (hFinalScore > gFinalScore) {
-    finalMsg.textContent = "The Winner is Team Home 🥳";
-  }
-  else if (hFinalScore < gFinalScore) {
-    finalMsg.textContent = "The Winner is Team Guest 🥳";
-  }
-  else {
-    finalMsg.textContent = "The Match is a Draw!";
-  }
+
+  const finalMessage = hFinalScore > gFinalScore ? "The Winner is Team Home 🥳"
+    : hFinalScore < gFinalScore ? "The Winner is Team Guest 🥳"
+    : "The Match is a Draw!"
```

Comment:
- [ ] This one is a nested ternary operator which increases the code complexity.
You can use it like 
```
let finalMessage = ""
if(hFinalScore < gFinalScore) {
 finalMessage = "The Match is a Draw!"
}

if (hFinalScore > gFinalScore) {
finalMessage = "The Winner is Team Home 🥳"
} else if (hFinalScore < gFinalScore) {
finalMessage = "The Winner is Team Guest 🥳"
}
```

Reply:
resolved